fe/src/utils/nav.ts

```diff
@@ -0,0 +1,10 @@
+export const NAV_ITEMS = [
```

Comment:
You'll have to export through an `index.ts` file to explicitly say it's an public export. Otherwise lint will be mad (and I will be as well)

Reply:
Yes, then you won't need to do point number 1 in my comment. 👍